frontend/tests/features/cellGuide/cellGuide.test.ts

```diff
@@ -482,19 +493,29 @@ describe("Cell Guide", () => {
             const dropdown = page.getByTestId(
               CELL_GUIDE_CARD_GLOBAL_TISSUE_FILTER_DROPDOWN
             );
+
             await waitForElementAndClick(dropdown);
             await page.keyboard.type("colon");
             await page.keyboard.press("ArrowDown");
             await page.keyboard.press("Enter");
 
-            const rowElementsAfter = await page
-              .locator(`${tableSelector} tbody tr`)
-              .all();
-            const rowCountAfter = rowElementsAfter.length;
-            expect(rowCountAfter).toBeGreaterThan(1);
-            const firstRowContentAfter =
-              await rowElementsAfter[0].textContent();
-            expect(firstRowContentBefore).not.toBe(firstRowContentAfter);
+            await tryUntil(
```

Comment:
update test to avoid flakiness caused by table not yet loaded new data for specific tissue